scripts/common/preflights.sh

```diff
@@ -423,18 +430,26 @@ function host_preflights() {
 
     logStep "Running host preflights"
     if [ "${PREFLIGHT_IGNORE}" = "1" ]; then
-        "${DIR}"/bin/kurl host preflight "${MERGED_YAML_SPEC}" ${opts} || true
+        "${DIR}"/bin/kurl host preflight "${MERGED_YAML_SPEC}" ${opts} > "${out_file}" || true
+
+        cat "${out_file}"
+        host_preflights_mkresults "${out_file}" "${opts}"
+
         # TODO: report preflight fail
     else
         # interactive terminal
         if prompts_can_prompt; then
             set +e
-            "${DIR}"/bin/kurl host preflight "${MERGED_YAML_SPEC}" ${opts} </dev/tty
+            "${DIR}"/bin/kurl host preflight "${MERGED_YAML_SPEC}" ${opts} > "${out_file}" </dev/tty
```

Comment:
should we use tee here instead?

Reply:
that would change the exit code. i can only tee on line 433 but then it would be inconsistent

Reply:
I'm fine with this since (discussed elsewhere) progress is on stderr and thus won't be interrupted, only results are on stdout